sotodlib/core/axisman_io.py

```diff
@@ -159,6 +159,13 @@ def _save_axisman(axisman, dest, group=None):
     else:
         f = None
     if group is not None:
+        if group in dest:
```

Comment:
> I definitely like adding the overwrite option but could we make it an option for group=None as well? If that's the case we could wipe the whole file. It makes sense to me to have overwrite work for all use cases.

Sure, will take a stab at this.  I think one needs to go to emptying the existing group instead of simply unlinking it ... but that has other potential benefits.